contracts/contracts/RequestManager.sol

```diff
@@ -252,15 +252,13 @@ contract RequestManager is Ownable {
 
         address nextActor;
         uint256 minValue;
-        uint256 periodExtension;
+        uint256 periodExtension = challengePeriodExtension;
         uint256 claimerStake = claim.claimerStake;
         uint256 challengerStakeTotal = claim.challengerStakeTotal;
 
         if (claimerStake > challengerStakeTotal) {
             if (challengerStakeTotal == 0) {
-                periodExtension = finalizationTimes[request.targetChainId] + challengePeriodExtension;
-            } else {
-                periodExtension = challengePeriodExtension;
+                periodExtension += finalizationTimes[request.targetChainId];
             }
             require(claim.claimer != msg.sender, "Cannot challenge own claim");
             nextActor = msg.sender;
```

Comment:
Nitpick: would be nice to squash this with the first commit that touches this part.